coralproject/cay
# Reader facing form design (Cay Side)

## What does this PR do?
Changes the default theme for forms. Issue: #413 

When merging this, its Elkhorn counterpart should be merged too.
See:  https://github.com/coralproject/elkhorn/pull/54

## How do I test this PR?
- Check the *reader-facing-form-design* branch on your local Elkhorn.
- Go to *Create form*
- Add every field type but Location 
- Add titles for all fields, tweak some options.
- Click on preview and compare to the design on the issue.
